src/main/java/thaumicboots/events/BootsEventHandler.java

```diff
@@ -349,8 +318,8 @@ else if ((item instanceof ItemElectricCometBoots) || (item instanceof ItemCometV
 
             itemStack.stackTagCompound.setInteger("runTicks", ticks);
         }
-
-        else if ((item instanceof ItemMeteoricCometBoots || (item instanceof ItemCometMeteorBoots))) {
+        // mixed boots
+        else if (item instanceof IMeteor) {
```

Comment:
@boubou19 as you can see it's just a trigger

Reply:
yeah but you start to implement them to other boots than Comet and Meteor, like the void walker one, so they lose their purpose of being able to identify the kind of boots they are

Reply:
Only the meteor and comet voidwalker ones

Reply:
done